Enterprise-CMCS/macpro-mdct-mcr
# MLR Submission Modal (Hydrate on Edit)

## Summary

### Description

Bug fix for MLR report submission modal not hydrating the submission name.

### Related ticket

N/A

### How to test

- Log in as a state user
- Create a new MLR report
- Edit that report name

Everything should work as expected ✨ 

### Important updates


### Author checklist
